pom.xml

```diff
@@ -50,6 +50,7 @@
         <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
         <project.reporting.outputEncoding>UTF-8</project.reporting.outputEncoding>
         <cassandra.version>3.11.2</cassandra.version>
+        <scylla.version/>
```

Comment:
I don't understand the purpose of this property.

Reply:
We use it to set scylla version, this change is only to put it into the pom to make it visible there.